config-connector/solutions/projects/kpt/project-services/README.md

```diff
@@ -0,0 +1,61 @@
+Project Services
+==================================================
+
+# NAME
+
+  project-services
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to enable services on a project.
+  
+## Consumption
+
+  Using kpt:
+  ```
+  BASE=https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+  kpt pkg get $BASE/config-connector/solutions/projects/kpt/project-services project-services
+  ```
+
+## Requirements
+
+  A working cluster with Config Connector installed.
+  
+  The cnrm-system service account must have
+`roles/serviceusage.serviceUsageAdmin` or `roles/owner` for the desired project.
+
+
+## Usage
+  If you are enabling services for a project other than the one you have
+installed Config Connector in, set the `project-id` and `namespace`:
+
+  ```
+  kpt cfg set . project-id VALUE
+  kpt cfg set . namespace project-annotated
```

Comment:
The customer shouldn't need to set the namespace.

Reply:
I made the namespace a setter and set the default to default so that if the customer is enabling services in the same project that they are running KCC from they won't need to set the project-id or the namespace.